env0/resource_environment_test.go

```diff
@@ -1329,16 +1329,16 @@ func TestUnitEnvironmentWithoutTemplateResource(t *testing.T) {
 	templateCreatePayload := client.TemplateCreatePayload{
 		Repository:           template.Repository,
 		Description:          template.Description,
-		GithubInstallationId: template.GithubInstallationId,
+		GithubInstallationId: client.NullInt(template.GithubInstallationId),
```

Comment:
can the types be imported once? IMO it's less elegant to use `client.TypeX` for casting every time